### ISSUE TITLE
use local version python compiled lizard.py but did not generate lizard

### ISSUE THREAD

**Issue by JiahangLi** [terryyin/lizard#139]
On my Linux account, I tried to installed newer version of python in order to run lizard due to I don't have the root access and root's version is too old to compile.

It seems lizard executable seemed not generated after the script ran successfully.

**Comment by terryyin** [comment]
Lizard requires Python 2.7 or Python 3.x. You can install lizard to your system via `pip install lizard` (need root access if you are not using a Python virtual env).

Or you can download/clone the source code and just run python lizard.py in the folder.

If you run `python setup.py install`, then the “executable” will be put to the global python script path. Just try `lizard` in your command line and see if you’ve got it already.

**Comment by JiahangLi** [comment]
Hey Terry,
Thanks for your feedback. The third solution fixed my problem in this case!

I'm working on integrate lizard with Jenkins right now. Is there a version requirement for Jenkins in order for it to display the lizard output plot? Or specific plug-ins to plot the graph, i.e JUnit or XUnit?

Due to the limited internet connection and old version of java, I'm using Jenkins1.5. Tried to use the JUnit to plot Jenkins .xml output but Jenkins failed with following output:
ERROR: Failed to archive test reports
hudson.util.IOException2: Failed to read 

Thanks!

**Comment by terryyin** [comment]
Have you added the -X option to output the xml?

**Comment by JiahangLi** [comment]
Yes. I did and I got the xml file. However, it seems that Jenkins can not interrupt the file correctly. 
I opened the xml file by using excel and other viewer, but the display is a bit different from what I thought it would be.

I might be wrong, but for my personal understanding, if I open the xml using excel the display would look like the table display on our online analyzer, right?

![capture](https://cloud.githubusercontent.com/assets/13212671/17335806/9eb22f60-5898-11e6-9788-fbabc8cee19c.PNG)

I really appreciate your help
